src/Sentry.AspNetCore/SentryTracingMiddleware.cs

```diff
@@ -44,6 +44,8 @@ public async Task InvokeAsync(HttpContext context)
             }
             finally
             {
+                transaction.Name = context.GetTransactionName();
```

Comment:
Let's add a comment here to explain why we get the transaction name again

Reply:
@Tyrrrz Not "again", if you check a few lines above I've replaced the default value with "Unknown Route"